src/utils.rs

```diff
@@ -4,6 +4,8 @@ use crate::{format::Format, scale::Scale};
 pub(crate) fn scale_format_to_string(scale: &Scale, format: &Format) -> String {
     match format {
         Format::BinarySI => match scale {
+            Scale::Nano => "n".to_owned(),
+            Scale::Micro => "u".to_owned(),
```

Comment:
Ah, like `Milli`, I suppose these should be `""`, given they aren't applicable?

Reply:
That's a good question.
I'll have to look into the kube source and how they are handling it.

In the current implementation in this crate, I assumed that Milli would be the smallest unit, and then I "upgraded" it to a simple decimal. However, considering there are smaller denominations, this might need to be changed, and the smaller units should be returned. (Or add modulo logic so that if the value modulo 1000 is 0, the next higher unit is used, and the number is divided by 1000).

Reply:
But this should not block this PR from getting merged, as from a mathematical point of view, the results are correct.